web/navbar.jsp

```diff
@@ -1,3 +1,4 @@
+<link href="images/logo.jpg" rel="icon">
```

Comment:
Since the navbar is a component and is being used in many places inside a web page, it is not necessary to add a favicon explicitly in this component. So, please remove the favicon from this component.